### PR TITLE
Add paramaeters related to Taint based Evictions in kube-apiserver

### PR DESCRIPTION
**Pull request by h3poteto**
closes: #6612 

I added two parameters in kube-apiserver: 

- defaultNotReadyTolerationSeconds
- defaultUnreachableTolerationSeconds

these parameters control the pod toleration time.
You can specify it in cluster spec:

```yaml
spec:
  # ...
  kubeAPIServer:
    defaultNotReadyTolerationSeconds: 600
    defaultUnreachableTolerationSeconds: 600
```

### CODE REVIEW

**Comment by hakman** [comment]
Thanks @h3poteto. Looks good. Would you mind also adding some mention of this to https://github.com/kubernetes/kops/blob/master/docs/cluster_spec.md#kubeapiserver?
/ok-to-test

**Comment by h3poteto** [comment]
@hakman Hi, thank you. I see, I will add it.

File: docs/cluster_spec.md [hunk 1]
```diff
@@ -446,6 +446,17 @@ spec:
     eventTTL: 03h0m0s
 ```
 
+### Toleration Seconds
+
+There are two parameters related to Taint based Evictions. These parameters indicate default value of the `tolerationSeconds` for `notReady:NoExecute` and `unreachable:NoExecute`.
```
Comment (hakman) [inline docs/cluster_spec.md:451]:
```suggestion
### Taint based Evictions

There are two parameters related to taint based evictions. These parameters indicate default value of the `tolerationSeconds` for `notReady:NoExecute` and `unreachable:NoExecute`.
```

Comment (hakman) [reply docs/cluster_spec.md:451]: Please update and squash the commits and I think this is pretty much all.

File: pkg/apis/kops/componentconfig.go [hunk 2]
```diff
@@ -490,6 +490,11 @@ type KubeAPIServerConfig struct {
 	// CorsAllowedOrigins is a list of origins for CORS. An allowed origin can be a regular
 	// expression to support subdomain matching. If this list is empty CORS will not be enabled.
 	CorsAllowedOrigins []string `json:"corsAllowedOrigins,omitempty" flag:"cors-allowed-origins"`
+
+	// DefaultNotReadyTolerationSeconds indicates the tolerationSeconds of the toleration for notReady:NoExecute that is added by default to every pod that does not already have such a toleration.
+	DefaultNotReadyTolerationSeconds *int32 `json:"defaultNotReadyTolerationSeconds,omitempty" flag:"default-not-ready-toleration-seconds"`
+	// DefaultUnreachableTolerationSeconds indicates the tolerationSeconds of the toleration for unreachable:NoExecute that is added by default to every pod that does not already have such a toleration.
+	DefaultUnreachableTolerationSeconds *int32 `json:"defaultUnreachableTolerationSeconds,omitempty" flag:"default-unreachable-toleration-seconds"`
```
Comment (hakman) [inline pkg/apis/kops/componentconfig.go:497]: Small nit, seems that k8s uses int64 here. Probably no reason to be so big, but should keep it the same:
https://github.com/kubernetes/kubernetes/blob/cea1d4e20b4a7886d8ff65f34c6d4f95efcb4742/plugin/pkg/admission/defaulttolerationseconds/admission.go#L35

```suggestion
	// DefaultNotReadyTolerationSeconds indicates the tolerationSeconds of the toleration for notReady:NoExecute that is added by default to every pod that does not already have such a toleration.
	DefaultNotReadyTolerationSeconds *int64 `json:"defaultNotReadyTolerationSeconds,omitempty" flag:"default-not-ready-toleration-seconds"`
	// DefaultUnreachableTolerationSeconds indicates the tolerationSeconds of the toleration for unreachable:NoExecute that is added by default to every pod that does not already have such a toleration.
	DefaultUnreachableTolerationSeconds *int64 `json:"defaultUnreachableTolerationSeconds,omitempty" flag:"default-unreachable-toleration-seconds"`
```

Comment (h3poteto) [reply pkg/apis/kops/componentconfig.go:497]: Oh, I see. I will fix it.

**Comment by hakman** [comment]
👍 
/lgtm
/approve

**Pull request merged**